BurstBotNET/Commands/ChaseThePig/Game.cs

```diff
@@ -0,0 +1,899 @@
+                            .Select(c =>
+                            {
+                                if (firstCard != null && c.Suit == firstCard.Suit && c.Number == firstCard.Number)
+                                {
+                                    var value = c.ToSpecifier() switch
+                                    {
+                                        "c10" => ChasePigExposure.FirstTransformer.ToString(),
+                                        "sq" => ChasePigExposure.FirstDoublePig.ToString(),
+                                        "dj" => ChasePigExposure.FirstDoubleGoat.ToString(),
+                                        "ha" => ChasePigExposure.FirstDoubleMinus.ToString(),
+                                        _ => string.Empty
+                                    };
+
+                                    return new SelectOption(c.ToStringSimple(), value, c.ToStringSimple(),
+                                        new PartialEmoji(c.Suit.ToSnowflake()));
+                                }
+
+                                var v = c.ToSpecifier() switch
+                                {
+                                    "c10" => ChasePigExposure.Transformer.ToString(),
+                                    "sq" => ChasePigExposure.DoublePig.ToString(),
+                                    "dj" => ChasePigExposure.DoubleGoat.ToString(),
+                                    "ha" => ChasePigExposure.DoubleMinus.ToString(),
+                                    _ => string.Empty
+                                };
+
+                                return new SelectOption(c.ToStringSimple(), v, c.ToStringSimple(),
+                                    new PartialEmoji(c.Suit.ToSnowflake()));
+                            }).ToImmutableArray();
+
+                        var components = new IMessageComponent[]
+                        {
+                            new ActionRowComponent(new[]
+                            {
+                                new SelectMenuComponent("chase_pig_expose_menu", selectOptions,
+                                    localization.ChaseThePig.Expose, 0, selectOptions.Length)
+                            }),
+                            new ActionRowComponent(new[]
+                            {
+                                new ButtonComponent(ButtonComponentStyle.Danger,
+                                    localization.ChaseThePig.NoExpose,
+                                    new PartialEmoji(Name: Constants.CrossMark),
+                                    "chase_pig_decline_expose"),
+                                new ButtonComponent(ButtonComponentStyle.Primary,
+                                    localization.ChaseThePig.ShowHelp,
+                                    new PartialEmoji(Name: Constants.QuestionMark),
+                                    "chase_pig_help")
+                            })
+                        };
```

Comment:
This can be moved out of the method too

Reply:
The second action row can be moved out, but when compared with line 338 to 349, there's an additional action row here. Do you think it would be better to just move out the second action row?

Reply:
I suppose keep it if it's going to make more lines when moving it